packages/monitoring/src/react/hooks.ts

```diff
@@ -5,6 +5,6 @@ import { MonitoringContext } from './context';
 /**
  * Custom hook that allows you to access the Monitoring Context methods inside a component.
  */
-export function hooks(): MonitoringAdapter {
+export function useMonitoring(): MonitoringAdapter {
```

Comment:
I have made these changes in my pr and it's now merged. So we can remove this one.